tasks/run.json

```diff
@@ -9,6 +9,10 @@
     "environment": {
       "description": "The desired puppet code environment to use",
       "type": "Optional[String[1]]"
+    },
+    "tags": {
+      "description": "Optional string that will be passed to --tags",
+      "type": "Optional[Array[String[1]]]"
```

Comment:
Puppet accepts a [comma-separated list of tags](https://www.puppet.com/docs/puppet/8/configuration.html#tags), so wondering if this should this be?

```ruby
Optional[Variant[String[1],Array[String[1]]]
```